client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -143,13 +97,13 @@ const PluginsBrowser = ( { trackPageViews = true, category, search, hideHeader }
 				popularSearchesRef={ searchHeaderRef }
 				isSticky={ isAboveElement }
 				searchTerm={ search }
-				isSearching={ isFetchingPluginsBySearchTerm }
+				isSearching={ false }
```

Comment:
Not sure what to do about this, need to take a closer look.

Reply:
There is a `temporary fix` [here](https://github.com/Automattic/wp-calypso/blob/2ba8bb7856dcca916ec7a982369c2d76c2c5148d/client/my-sites/plugins/plugins-browser/index.jsx#L57)

Reply:
I fixed it in the new commit. I use the Component instead of an Element in the controller and then render it on PluginBrowser.